modules/dynamic-plugins/rhdh-supported-plugins.csv

```diff
@@ -1,80 +1,80 @@
-"Signals ","@backstage/plugin-signals","Frontend","0.0.19","Red Hat Tech Preview","./dynamic-plugins/dist/backstage-plugin-signals",";","Disabled"
-"SonarQube ","@backstage-community/plugin-sonarqube","Frontend","0.13.0","Red Hat Tech Preview","./dynamic-plugins/dist/backstage-community-plugin-sonarqube",";","Disabled"
-"SonarQube ","@backstage-community/plugin-sonarqube-backend","Backend","0.9.2","Red Hat Tech Preview","./dynamic-plugins/dist/backstage-community-plugin-sonarqube-backend-dynamic","`SONARQUBE_URL`;`SONARQUBE_TOKEN`;","Disabled"
-"SonarQube ","@backstage-community/plugin-scaffolder-backend-module-sonarqube","Backend","2.7.1","Red Hat Tech Preview","./dynamic-plugins/dist/backstage-community-plugin-scaffolder-backend-module-sonarqube-dynamic",";","Disabled"
-"Tech Radar ","@backstage-community/plugin-tech-radar","Frontend","1.7.0","Red Hat Tech Preview","./dynamic-plugins/dist/backstage-community-plugin-tech-radar",";","Disabled"
-"Tech Radar ","@backstage-community/plugin-tech-radar-backend","Backend","1.6.0","Red Hat Tech Preview","./dynamic-plugins/dist/backstage-community-plugin-tech-radar-backend-dynamic","`TECH_RADAR_DATA_URL`;","Disabled"
-"Utils ","@roadiehq/scaffolder-backend-module-utils","Backend","3.5.0","Red Hat Tech Preview","./dynamic-plugins/dist/roadiehq-scaffolder-backend-module-utils-dynamic",";","Disabled"
+"Name","Plugin","Role","Version","Support Level","Lifecycle","Path","Required Variables","Default"
+"Adoption Insights","@red-hat-developer-hub/backstage-plugin-adoption-insights","Frontend","0.2.1","Production","active","./dynamic-plugins/dist/red-hat-developer-hub-backstage-plugin-adoption-insights",";","Enabled"
+"Adoption Insights","@red-hat-developer-hub/backstage-plugin-adoption-insights-backend","Backend","0.2.1","Production","active","./dynamic-plugins/dist/red-hat-developer-hub-backstage-plugin-adoption-insights-backend-dynamic",";","Enabled"
+"Analytics Module Adoption Insights","@red-hat-developer-hub/backstage-plugin-analytics-module-adoption-insights","Frontend","0.2.0","Production","active","./dynamic-plugins/dist/red-hat-developer-hub-backstage-plugin-analytics-module-adoption-insights-dynamic",";","Enabled"
+"Analytics Provider Segment","@backstage-community/plugin-analytics-provider-segment","Frontend","1.16.0","Production","active","./dynamic-plugins/dist/backstage-community-plugin-analytics-provider-segment","`SEGMENT_TEST_MODE`;`SEGMENT_WRITE_KEY`;","Enabled"
+"Argo CD","@roadiehq/backstage-plugin-argo-cd-backend","Backend","4.3.1","Production","active","./dynamic-plugins/dist/roadiehq-backstage-plugin-argo-cd-backend-dynamic","`ARGOCD_AUTH_TOKEN`;`ARGOCD_AUTH_TOKEN2`;`ARGOCD_INSTANCE1_URL`;`ARGOCD_INSTANCE2_URL`;`ARGOCD_PASSWORD`;`ARGOCD_USERNAME`;","Disabled"
```

Comment:
does it matter that these variables are now sorted differently? wondering about UX ...